examples/src/main/scala/sttp/tapir/examples/PrometheusMetricsExample.scala

```diff
@@ -0,0 +1,89 @@
+package sttp.tapir.examples
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Directives.concat
+import akka.http.scaladsl.server.Route
+import com.typesafe.scalalogging.StrictLogging
+import io.circe.generic.auto._
+import io.prometheus.client.{CollectorRegistry, Counter}
+import sttp.tapir._
+import sttp.tapir.generic.auto._
+import sttp.tapir.json.circe._
+import sttp.tapir.metrics.prometheus.PrometheusMetrics
+import sttp.tapir.metrics.{EndpointMetric, Metric}
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.server.akkahttp.{AkkaHttpServerInterpreter, AkkaHttpServerOptions}
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Future}
+
+object PrometheusMetricsExample extends App with StrictLogging {
+
+  case class Person(name: String)
+
+  // Simple endpoint returning 200 or 400 response with string body
+  val personEndpoint: ServerEndpoint[Person, String, String, Any, Future] =
+    endpoint.post
+      .in("person")
+      .in(jsonBody[Person])
+      .out(stringBody)
+      .errorOut(stringBody)
+      .serverLogic { p => Future.successful { if (p.name == "Jacob") Right("Welcome") else Left("Unauthorized") } }
+
+  val collectorRegistry = CollectorRegistry.defaultRegistry
+
+  // Metric for counting responses labeled by path, method and status code
+  val responsesTotal = Metric[Future, Counter](
```

Comment:
I think in the example we should show how to use default metrics, and move the custom metric creation to the docs